BraveWallet/Crypto/BuySwapSwap/SwapCryptoView.swift

```diff
@@ -170,16 +393,28 @@ struct SwapCryptoView: View {
           }
         }
       }
+      .onAppear {
+        swapTokensStore.prepare(with: keyringStore.selectedAccount)
+      }
     }
   }
+  
+  private func formatSlippage() -> String {
+    let nf = NumberFormatter()
+    nf.numberStyle = .percent
+    nf.maximumFractionDigits = 2
+    
+    return nf.string(from: NSNumber(value: slippage)) ?? ""
+  }
```

Comment:
Doesn't need to be a function

```suggestion
  var formattedSlippage: String {
    let nf = NumberFormatter()
    nf.numberStyle = .percent
    nf.maximumFractionDigits = 2
    return nf.string(from: NSNumber(value: slippage)) ?? ""
  }
```